packages/patternfly-4/react-core/src/components/Select/selectConstants.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export const SelectContext = React.createContext({
+  onSelect: () => {},
+  onClose: () => {}
+});
+
+export const SelectVariant = {
+  single: 'single'
+};
+
+export const KeyTypes = {
+  Tab: 'Tab',
```

Comment:
Can you reuse the constants in the helpers/constants file here?

Reply:
Sure to both.